entity-framework/core/modeling/data-seeding.md

```diff
@@ -5,38 +5,74 @@ author: AndriySvyryd
 ms.date: 11/02/2018
 uid: core/modeling/data-seeding
 ---
+
 # Data Seeding
 
 Data seeding is the process of populating a database with an initial set of data.
 
 There are several ways this can be accomplished in EF Core:
 
-* Model seed data
-* Manual migration customization
+* Configuration options data seeding (`UseSeeding`)
 * Custom initialization logic
+* Model managed data (`HasData`)
+* Manual migration customization
+
+<a name="use-seeding-method"></a>
+
+## Configuration options `UseSeeding` and `UseAsyncSeeding` methods
+
+EF 9 introduced `UseSeeding` and `UseAsyncSeeding` methods, which provide a convenient way of seeding the database with initial data. These methods aim to improve the experience of using custom initialization logic (explained below). They provide one clear location where all the data seeding code can be placed. Moreover, the code inside `UseSeeding` and `UseAsyncSeeding` methods is protected by the [migration locking mechanism](/ef/core/what-is-new/ef-core-9.0/whatsnew#concurrent-migrations) to prevent concurrency issues. The new seeding methods are called as part of `EnsureCreated` operation, `DbContext.Database.Migrate()` and `dotnet ef database update` command, even if there are no model changes and no migrations were applied.
```

Comment:
```suggestion
EF 9 introduced `UseSeeding` and `UseAsyncSeeding` methods, which provide a convenient way of seeding the database with initial data. These methods aim to improve the experience of using custom initialization logic (explained below). They provide one clear location where all the data seeding code can be placed. Moreover, the code inside `UseSeeding` and `UseAsyncSeeding` methods is protected by the [migration locking mechanism](/ef/core/what-is-new/ef-core-9.0/whatsnew#concurrent-migrations) to prevent concurrency issues and are now the recommended way to add initial data in your app.

The new seeding methods are called as part of `<xref:Microsoft.EntityFrameworkCore.Storage.IDatabaseCreator.EnsureCreated>` operation, `<xref:Microsoft.EntityFrameworkCore.RelationalDatabaseFacadeExtensions.Migrate>` and `dotnet ef database update` command, even if there are no model changes and no migrations were applied.
```